library/swift/src/RetryPolicyMapper.swift

```diff
@@ -36,8 +36,10 @@ extension RetryPolicy {
     self.init(
       maxRetryCount: maxRetryCount,
       retryOn: headers.value(forName: "x-envoy-retry-on")?.compactMap(RetryRule.init) ?? [],
+      // TODO: does the UInt.init blow up? or does it return nil?
```

Comment:
@rebello95 Could you give insight onto how we can find this out (or help find this out for us?)

Reply:
Just checked, and mapping `String` to `UInt*` here just returns `nil` if it's not a valid unsigned int